controllers/util/misc.go

```diff
@@ -72,6 +72,24 @@ func AddLabel(obj client.Object, key, value string) bool {
 	return !labelAdded
 }
 
+func AddAnnotation(obj client.Object, key, value string) bool {
```

Comment:
The Ramen codebase currently contains multiple versions of the `AddAnnotation()` function. This is a good start to consolidate them in util/misc.go, but we need to clean up the others to use this one instead. Could be a future task.

Reply:
Noted here https://github.com/RamenDR/ramen/issues/1097